eng/pipelines/corefx-base.yml

```diff
@@ -84,36 +84,28 @@ jobs:
           - group: DotNet-Internal-Tools-Feed
           - group: DotNet-Blob-Feed
 
-        - ${{ if eq(parameters.isOfficialBuild, 'false') }}:
-          - _msbuildCommonParameters: ''
-
         - ${{ if eq(job.submitToHelix, 'true') }}:
           - group: DotNet-HelixApi-Access
           - _archiveTestsParameter: /p:ArchiveTests=Tests
 
+        - _args: -restore -build -configuration $(_BuildConfig) -ci -buildtests -arch $(_architecture) -framework $(_framework) $(_archiveTestsParameter)
```

Comment:
Can we do this multiline? for readability purpose? Just like the call to the build script later on this file

Reply:
sure, if that's the only nit I would prefer do that in a follow-up PR (i.e. codecoverage)